kong/dao/cassandra_db.lua

```diff
@@ -62,7 +62,7 @@ local function serialize_arg(field, value)
     return cassandra.uuid(value)
   elseif field.type == "timestamp" then
     return cassandra.timestamp(value)
-  elseif field.type == "table" then
+  elseif field.type == "table" or field.type == "array" then
```

Comment:
@PGBI I am wondering if this is really necessary, since we have other cases where we are using arrays and it already works perfectly (ie, https://github.com/Mashape/kong/blob/master/kong/plugins/acl/schema.lua#L7).